docs/managing-dragonfly/managing-dragonfly.md

```diff
@@ -3,3 +3,14 @@ sidebar_position: 5
 ---
 
 # Managing Dragonfly
+
+### Limits
+
+Listed below are known Dragonfly limitations.
+
+* Each Dragonfly instance can handle up to two billion keys
+* Lists, Sets and Hashes can contain up to two billion values
```

Comment:
B+ tree and redis structures are limited by uint32, denseset is theoretically able to hold more but it doesn't matter. Quote redis: `In other words your limit is likely the available memory in your system`